LostCities/Game.py

```diff
@@ -1,67 +1,94 @@
+import os
+
 from Discardpile import Discardpile
 from Drawpile import Drawpile
 from Player import Player
 from Playercardpile import Playercardpile
 
 
 class Game:
-    def __init__(self) -> None:
-        """The game class. Here every other class will be made and the game itself will be written
-
-
-        Member variables:
-            self.Player1 (Player): player1
-            self.Player2 (Player): player2
-            self.MAXROUNDS (int): 3 max rounds in game
-            self.current_round (int): current round variable
-
-
+    def __init__(
+        self,
+        colors: list[str],
+        MAXROUNDS: int,
+        current_round: int,
+        Player1: Player,
+        Player2: Player,
+        player1_piles: list[Playercardpile],
+        player2_piles: list[Playercardpile],
+        current_player: Player,
+        current_player_piles: list[Playercardpile],
+        drawpile: Drawpile,
+        discardpiles: list[Discardpile],
```

Comment:
Ezeket miért kívülről kapja, paraméterként? Pont a konstruktor dolga lenne inicializálni a saját tagváltozóit. Ha a használójának kell megadnia ezeket, akkor értelmét veszti a konstruktor.

Reply:
Valamikor írtad a member variable-ek nél hogy ne úgy írjak be mindent mert úgy ki kell keresni őket egyesével, és azt hittem hogy akkor írjam be őket inkább a konstruktorba, de akkor visszacsinálom

Reply:
Igen, hogy legyen inicializálva meg dokumentálva a konstruktorban, de nem kell paraméterben bekérni az értékét.